app/code/Magento/Fedex/Model/Carrier.php

```diff
@@ -1571,7 +1571,7 @@ private function processTrackingDetails(\stdClass $trackInfo)
         ];
 
         if (!empty($trackInfo->ShipTimestamp)) {
-            $datetime = \DateTime::createFromFormat(\DateTime::ISO8601, $trackInfo->ShipTimestamp);
+            $datetime = \DateTime::createFromFormat('U', strtotime($trackInfo->ShipTimestamp));
```

Comment:
If a timezone is not specified, the time will be parsed according to configured timezone for Magento store, but I think it's more correctly to parse time in that case with UTC timezone.